static/app/utils/customMeasurements/customMeasurements.tsx

```diff
@@ -0,0 +1,7 @@
+type CustomMeasurement = {
```

Comment:
I'm wondering if we should tie `Measurement` and `CustomMeasurements` together. Like,
```ts
type CustomMeasurement = Measurement & {
  supportedFunctions: string[];
}
```

if we expect `CustomMeasurement` to be a superset of `Measurement` values. But since we only have `key` and `name` I guess it's not a big deal at this point

Reply:
Good point, I've updated CustomMeasurement to superset Measurement!